doc/ref_arch/kubernetes/chapters/chapter06.md

```diff
@@ -23,20 +23,20 @@
 
 <a name="6.1"></a>
 ## 6.1 Introduction
-Securing Kubernetes requires several layers of security features to provide end to end security for cloud native applications. It is also important to adopt a layered packaging model which support separation of concerns during image build. A fully integrated security testing should be baked into the CI/CD pipeline. Automated security policies should also be used to flag builds with issue.  Image registries must be monitored to automatically block or replace images with known vulnerabilites, while also ensuring policies are used to gate what can be deployed and who can deploy from the registry.
+Securing Kubernetes requires several layers of security features to provide end to end security for cloud native applications. It is also important to adopt a layered packaging model which support separation of concerns during image build. A fully integrated security testing should be baked into the CI/CD pipeline. Automated security policies should also be used to flag builds with issues.  Image registries must be monitored to automatically block or replace images with known vulnerabilites, while also ensuring policies are used to gate what can be deployed and who can deploy from the registry.
```

Comment:
typo: "It is also important to adopt a layered packaging model which _**support**_ separation of concerns ..." ---> "... It is also important to adopt a layered packaging model which _**supports**_ separation of concerns..."

"A fully integrated security testing should be baked into the CI/CD pipeline." ---> "A fully integrated security testing _**phase (or stage)**_ should be baked into the CI/CD pipeline."